Gemfile.lock

```diff
@@ -62,15 +64,21 @@ GEM
     rspec-support (3.1.2)
     safe_yaml (1.0.4)
     slop (3.6.0)
+    slyphon-log4j (1.2.15)
```

Comment:
So weirdly enough when I run bundle install on your branch I don't pick up these changes- the only delta for me is `timecop (0.8.1)`.

 json is `json (1.8.3)` and not `json (1.8.3-java)`. What's up with that and `slyphon`?